static/app/types/alerts.tsx

```diff
@@ -28,11 +29,15 @@ export type IssueAlertRuleActionTemplate = {
   prompt: string;
   enabled: boolean;
   actionType?: 'ticket' | 'sentryapp';
-  formFields?: {
-    [key: string]: IssueAlertRuleFormField;
-  };
+  formFields?:
+    | {
+        [key: string]: IssueAlertRuleFormField;
+      }
+    | Config;
   ticketType?: string;
   link?: string;
+  uuid?: string;
+  sentryAppInstallationUuid?: string;
```

Comment:
@leeandher `sentryAppInstallationUuid` should not be undefined, we'll need it for the POST endpoint

Reply:
This is only because both the `ticketRules` and `sentryAppRules` are coerced into the same types. `ticketRules` won't have a `sentryAppInstallationUuid`